lib/plugins/plugin-validator.spec.js

```diff
@@ -13,6 +13,13 @@ describe('plugin-validator', () => {
   const validSuccessMessages = phaseOneSuccessMessages.concat([ansiColors.green('The plugin config is valid.')])
 
   beforeEach(() => {
+    // Polyfill process.exitCode for Node.js < v20
+    if (!process.exitCode) {
+      Object.defineProperty(process, 'exitCode', {
```

Comment:
Do you mind resetting to defaults afterwards?

```patch
afterEach(() => {
+ process.exitCode = undefined
  jest.clearAllMocks()
})
```

Probably won't affect the current tests, but good to reset our modifications

Reply:
Yes, done